Tasks/Melchikova/task3/task3_10.py

```diff
@@ -0,0 +1,9 @@
+a = int(input("Введите число: "))
+b = 0
+for i in range(2, a // 2+1):
+    if (a % i == 0):
+        b = b+1
+if (b <= 0):
+    print("Число простое")
+else:
+    print("Число не является простым")
```

Comment:
the task was to select a largest prime number from a list, not detect if any number is prime, -5 points for the incomplete task